### PR TITLE
Validation Prior to Deploy

### PR DESCRIPTION
**Pull request by positiveVibes4all**
Validation issue prevented deploy when merged, this should be added to the feature pipeline also but without an actual deploy so this can be caught prior in the future.

No version bump needed, because it was never deployed.

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>The version was not updated. Please update the version.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>The changelog has not been updated. Please update the changelog.</td>
    </tr>
  
<tr>
      <td>:no_entry_sign:</td>
      <td>Please assign someone to merge this PR, and optionally include people who should review.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>We were able to automatically fix some formatting issues in this PR for you!</td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


### Some things that were possibly fixed:

- Code that could be fixed via the --fix flag
- Formatting that could be fixed by prettier

Take a look at this commit to see what happened in detail: acaf20a5cd4fe34eae3277e15558b938fa0d28ee


And look at this wiki page to see the reasoning behind the ESLint rules: https://github.com/smartprocure/eslint-config-smartprocure/wiki/Rules-and-Why-We-Chose-Them
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4538b4bf00dff96f64beba96f0767e3196e31119
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+# 1.71.8
+
+- Corrected CI/CD issue
+
+# 1.71.7
+
+- Corrected CI/CD issue
```
Comment (stellarhoof) [inline CHANGELOG.md:7]: Duplicate change here

Comment (positiveVibes4all) [reply CHANGELOG.md:7]: There is technically already a 1.71.7 published, the pipeline failed to publish the doc site last and as such I felt like I should repeat the description here. I also have a note to fix the pipeline for this validation to be caught earlier. Here is the link the merge to master that I am talking about: https://app.circleci.com/pipelines/github/smartprocure/futil-js/334/workflows/d9a6c488-59cf-4661-9ea6-3be38a5e4308/jobs/1921

Let me know if there is some other way to handle this in your mind.

Comment (positiveVibes4all) [reply CHANGELOG.md:7]: Updated last comment, will fix this.

Comment (positiveVibes4all) [reply CHANGELOG.md]: Duplicates fixed

**Review by stellarhoof — APPROVED** [review]

**Pull request merged**